### PR TITLE
add DataHourMC command

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: docs/commands/uplink/DataHourMC.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+| ------------ | ------------------------------------------------------------ | ---------------------------------------------------------------- |
+| `1`          | `byte`                                                       | command id = `0x16`                                              |
+| `1`          | `byte`                                                       | command size (dynamic, `4+`)                                     |
+| `2`          | [packed date](../../types.md#packed-date)                    | [date](#date)                                                    |
+| `1`          | [packed hours](../../types.md#packed-hours)                  | [hours](#hours)                                                  |
+| `1..5`       | [extended value](../../types.md#extended-value)              | [channels bit set](#channels-bit-set)                            |
+| `[1..5]*n*h` | sequence of [extended values](../../types.md#extended-value) | [channel values with diffs sequence](#channel-values-with-diffs) |
+
+> `n` - amount of selected channels. <br>
+> `h` - amount of diffs for each channel.
```
Comment (DeadByElpy) [inline docs/commands/uplink/DataHourMC.md:21]:
```md
> `h` - amount of hours.
```
or

```md
> `h` - amount of hourly diffs for each channel.
```

?

File: docs/commands/uplink/DataHourMC.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **date**
+
+The command contains channel values data for this date.
```
Comment (DeadByElpy) [inline docs/commands/uplink/DataHourMC.md:29]:
```suggestion
The command contains pulse counter of channels for this date.
```
?

File: docs/commands/uplink/DataHourMC.md (hunk 1 above)
Comment (DarkPark) [reply docs/commands/uplink/DataHourMC.md]: applied

File: docs/commands/uplink/DataHourMC.md (hunk 2 above)
Comment (DarkPark) [reply docs/commands/uplink/DataHourMC.md]: applied

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**